services/core-api/app/api/services/email_service.py

```diff
@@ -192,6 +192,9 @@ def send_email(cls,
             current_app.logger.error(message)
             return
 
+        current_app.logger.info(
```

Comment:
Do we need _both_ debug and info?

Reply:
Changed from debug to info. And when we are using the templates its not possible to add the recipients list to the email body (for testing except prod). Hence just changed the existing debug log to in info.